### PR TITLE
Web modeler - Remove mentions of Web Modeler being Enterprise only

### PR DESCRIPTION
**Pull request by JacobiCamunda**
## Description

Part of https://github.com/camunda/web-modeler/issues/9111.

- Removes Enterprise only mentions
- Adds a link to the Docker setup repository
- Removes mentions that Web Modeler is only accessible for enterprise customers

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/self-managed/setup/deploy/other/docker.md [hunk 1]
```diff
@@ -8,21 +8,22 @@ keywords: ["camunda docker"]
 While the Docker images themselves are supported for production usage, [Docker Compose](/self-managed/setup/deploy/local/docker-compose.md) files are designed to be used by developers to run an environment locally; they are not designed to be used in production. We recommend to use [Kubernetes](/self-managed/setup/install.md) in production.
 :::
 
-We provide Docker images [via Dockerhub](https://hub.docker.com/u/camunda). All these images are publicly accessible (except for [Web Modeler](#web-modeler)).
+We provide Docker images [via Dockerhub](https://hub.docker.com/u/camunda). All these images are publicly accessible. Web Modeler consists of three components: `restapi`, `webapp`, and `websockets` which are publicly accessible for versions `>=8.6.0-alpha2` and for enterprise customers only all versions before.
```
Comment (wollefitz) [inline docs/self-managed/setup/deploy/other/docker.md:11]: ❓ Do we have to mention this at all here? 

The documentation is versioned so users who want to install `8.5` would directly access the `8.5` section of the documentation still stating that Web Modeler is not publicly available. For `next`, I'd say we don't have to explicitly mention that `8.6-alpha1` is not publicly available - the need for installing old alpha versions should be very low.

File: docs/self-managed/setup/deploy/other/docker.md [hunk 2]
```diff
@@ -49,11 +50,7 @@ For Web Modeler, we only provide multi-platform images from the following releas
 
 ### Web Modeler
 
-:::note
-Web Modeler Self-Managed is available to [enterprise customers](/reference/licenses.md#web-modeler) only.
-:::
-
-The Docker images for Web Modeler are not publicly accessible, but available to enterprise customers only from
+Docker images `<8.6.0-alpha2` for Web Modeler are not publicly accessible, but available to enterprise customers only from
```
Comment (wollefitz) [inline docs/self-managed/setup/deploy/other/docker.md:53]: Same as https://github.com/camunda/camunda-docs/pull/3887/files#r1627894167: I don't think we explicitly have to mention this anymore.

File: docs/self-managed/setup/deploy/other/docker.md [hunk 3]
```diff
@@ -62,7 +59,7 @@ Camunda's private Docker registry.
 | Frontend (`webapp`)   | `registry.camunda.cloud/web-modeler-ee/modeler-webapp:latest`     |
 | WebSocket server      | `registry.camunda.cloud/web-modeler-ee/modeler-websockets:latest` |
 
-To pull the images you first need to log in using the credentials you received from Camunda:
+If you pull the images from the private repository you first need to log in using the credentials you received from Camunda:
```
Comment (wollefitz) [inline docs/self-managed/setup/deploy/other/docker.md:62]: 🔧 Let's remove this (including the table above)

File: docs/self-managed/setup/deploy/other/docker.md
Comment (wollefitz) [inline docs/self-managed/setup/deploy/other/docker.md:1]: Let's move the sentence "You can also find more information on the supported [configuration variables](/self-managed/modeler/web-modeler/configuration/configuration.md)." below a new "Web Modeler" section below "Configuration hints"

File: docs/self-managed/setup/deploy/other/docker.md [hunk 4]
```diff
@@ -218,3 +193,7 @@ You can also add a Connector JAR using volumes:
 ```shell
 docker run --rm --name=connectors -d -v $PWD/connector.jar:/opt/app/ camunda/connectors:x.y.z
 ```
+
+### Web Modeler
+
+You can also find more information on the supported [configuration variables](/self-managed/modeler/web-modeler/configuration/configuration.md).
```
[the remark this answers is not in the document]
Comment (JacobiCamunda) [reply docs/self-managed/setup/deploy/other/docker.md:199]: What is even the difference?

Comment (wollefitz) [inline docs/self-managed/setup/deploy/other/docker.md:199]: I know I previously suggested this but reading through the page now this section seems unnecessary - let's just completely remove it.

**Comment by wollefitz** [comment]
Looks good to me now (apart from https://github.com/camunda/camunda-docs/pull/3887#discussion_r1628888929) - assigning @christinaausley for final sign-off.

**Comment by JacobiCamunda** [comment]
@wollefitz, I added a commit to also refer to the new license: 8e5a17d

**Review by conceptualshark — DISMISSED** [review]

File: docs/reference/licenses.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ The Connector SDK, REST Connector, Connector Runtime Docker image, and Connector
 
 ### Web Modeler
 
-Web Modeler is licensed under the proprietary Camunda Self-Managed Enterprise Edition license (a copy you obtain when you contact Camunda).
+Web Modeler is licensed under [Camunda License 1.0](https://legal.camunda.com/#camunda-license)
```
Comment (wollefitz) [inline docs/reference/licenses.md:25]:
```suggestion
Web Modeler is licensed under the [Camunda License 1.0](https://legal.camunda.com/#camunda-license).
```

**Review by christinaausley — APPROVED** [review]

**Pull request merged**